docs/pages/dsHDMIIn/ds-Hdmi-In_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,216 @@
+|4|vts_DolbyVision_stream|Format: Dolby Vision,Resolution: 3840 x 2160 (4K UHD),Color Depth: 10/12-bit,Color Space: Rec. 2020format and dynamic metadata.|
+|5|vts_HDR10plus_stream|Format: HDR10,Resolution: 3840 x 2160 (4K UHD),Color Depth: 10-bit,Color Space: Rec. 2020 and dynamic metadata capabilities.|
+
+Each test case need to verify with the each HdmiIn port.
+Below are top test use-case for the HdmiIn port.
+
+|#|Test-case|Description|Focus APIs|Source|Sink|
+|-|---------|-----------|----------|------|----|
+|1|Verify the HdmiIn Connect status with callback|Connect or Disconnect Hdmi input device on each of the hdmi input ports and check the hdmiInConnectCB callback is triggered|`dsHdmiInRegisterConnectCB()`|`Y`|`Y`|
+|2|Verify the HdmiIn Signal change with callback|Select the HdmiInput port and check the callback is triggered when the change in signal status occurs|`dsHdmiInRegisterSignalChangeCB()`|`NA`|`Y`|
+|3|Verify the HdmiIn Status change with callback|Select the HdmiInput port and check the callback is triggered when the status change occurs(i.e like isPresented, activeport) and  check the callbacks is triggered|`dsHdmiInRegisterStatusChangeCB()`|`NA`|`Y`|
+|4|Verify the HdmiIn Video Mode update  with callback|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,`DolbyVision`,`SDR`,...) and check the callbacks is triggered when we change the resolution|`dsHdmiInRegisterVideoModeUpdateCB()`|`NA`|`Y`|
+|5|Verify the HdmiIn Allm change  with callback|Change the allm option to TRUE/FALSE after connecting game controller on 4k supported panel and check the callbacks is triggered when the allm status change occurs|`dsHdmiInRegisterAllmChangeCB()`|`NA`|`Y`|
+|6|Verify the  HdmiIn Audio Video lateny with callback|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,`DolbyVision`,`SDR`,...) and check the callbacks is triggered when there is latency change|`dsHdmiInRegisterAVLatencyChangeCB()`|`NA`|`Y`|
+|7|Verify the HdmiIn AVI content change with callbacks|Play the streams from hdminput device video content with any format(like `HDR`,`HLG`,`DolbyVision`,`SDR`,...) and check the callbacks is triggered there is AVI content change|dsHdmiInRegisterAviContentTypeChangeCB()|`NA`|`Y`|
+|8|Get HdmiIn status of the selected port|Select the Hdmi Input and verify the status whether selected port active or not|`dsHdmiInSelectPort()`,`dsHdmiInGetStatus()`|`Y`|`Y`|
+|9|Scale HdmiIn video of the selected port|Select the Hdmi Input and scale the video on the selcted port verify video scaled or not|`dsHdmiInScaleVideo()`|`Y`|`Y`|
+|10|Check and verify Zoom mode selected|Select the Zoom mode from the available inputs and verify its set or not|`dsHdmiInSelectZoomMode()`|`Y`|`NA`|
+|11|Get Current video mode of slected input port|Play the video from the selected Hdmi Input port and get the current video mode|`dsHdmiInGetCurrentVideoMode()`|`Y`|`Y`|
```

Comment:
Does dsHdmiInGetCurrentVideoMode require to check with different streams?

Reply:
no , any one or two formats sufficient to confirm ,

Reply:
done

Reply:
Describe what are the formats required to use